TwilioVerifySDKTests/TwilioSecurity/Sources/Keychain/KeychainQueryTests.swift

```diff
@@ -76,7 +76,7 @@ class KeychainQueryTests: XCTestCase {
   
   func testSaveData_withKey_shouldReturnValidQuery() {
     let expectedData = "data".data(using: .utf8)!
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/a8a0ffb2344905f9a27850246dcbbe000db39834/TwilioVerifySDKTests/TwilioSecurity/Sources/Keychain/KeychainQueryTests.swift#L78"></span>Prefer non-optional <code>Data(_:)</code> initializer when converting <code>String</code> to <code>Data</code><br />
<code>non_optional_string_data_conversion</code> <code>KeychainQueryTests.swift:78</code></td>
    </tr>
  </tbody>
</table>